CMakeLists.txt

```diff
@@ -40,6 +40,15 @@ elseif(WIN32)
   option(USE_EXTERNAL_TINYXML2 "Use a system-installed version of tinyxml2" OFF)
 endif()
 
+#--------------------------------------
+# Option: Should we skip tests of deprecated functionality 
+option(SKIP_DEPRECATED_TESTS "Skip testing deprecated funcationality" OFF)
```

Comment:
Reading the description, I would have a question: is this skipping the build of the deprecated tests or is this skipping the run of the deprecated tests?

Reply:
> Reading the description, I would have a question: is this skipping the build of the deprecated tests or is this skipping the run of the deprecated tests?

The idea is to skip both.  This would primarily be for local development as building/running those tests isn't particularly valuable when you are iterating quickly.